build/templates/_converters.py.mako

```diff
@@ -239,7 +239,7 @@ def _convert_timedelta(value, library_type, scaling):
     if library_type in [_visatype.ViInt64, _visatype.ViInt32, _visatype.ViUInt32, _visatype.ViInt16, _visatype.ViUInt16, _visatype.ViInt8]:
```

Comment:
Should we stop mentioning ctypes if we no longer return a ctype?

Reply:
I considered that. Changing the comment to something like "convert float to int if an int is requested" would make it more generic. But on the other hand, it _is_ specifically because of ctypes that we do the conversion. So the comment as-is correctly explains the "why". If you think I should change it then I will, but so far I'm inclined to leave it alone.